packages/cookie-parser/index.ts

```diff
@@ -15,11 +15,11 @@ export class CookieParserMiddleware implements NestMiddleware {
     private static secret: string | string[];
     private static options: cookieParser.CookieParseOptions;
 
-    public resolve(...args: any[]): MiddlewareFunction {
+    public use(req: any, res: any, next: any) {
```

Comment:
i haven't used Nest in a long time, but how can this work? Seems like you should return a function to be passed into the main tree of middlewares. This seems like it would do nothing since you're just calling a function.

Reply:
This is the new standard for the internal function called in the middleware chain.
https://docs.nestjs.com/migration-guide